### PR TITLE
config: faster confirmation link expiration

### PR DESCRIPTION
**Pull request by topless**
* Fixed test where endpoint was registered after the first request
* Cleanup unused declarations
* Updated assets build for example app
* Tested expiration links for password recovery and email verification
* Closes #269

### CODE REVIEW

File: examples/app-setup.sh [hunk 1]
```diff
@@ -11,22 +11,10 @@ DIR=`dirname "$0"`
 cd $DIR
 export FLASK_APP=app.py
 
-# Setup app
-mkdir $DIR/instance
-
-mkdir static
-
-# Install specific dependencies
-pip install -r requirements.txt
```
Comment (lnielsen) [inline examples/app-setup.sh]: I think you need to leave this one

File: invenio_accounts/config.py [hunk 2]
```diff
@@ -90,6 +90,20 @@
 SECURITY_PASSWORD_SALT = None
 """Salt for storing passwords."""
 
+SECURITY_CONFIRM_EMAIL_WITHIN = '30 minutes'
```
Comment (lnielsen) [inline invenio_accounts/config.py]: Is it possible to add a test to ensure that the link expires

File: examples/app-setup.sh (hunk 1 above)
Comment (topless) [reply examples/app-setup.sh]: I have a branch to test it on my travis where I have reverted these ones.

**Comment by topless** [comment]
This pr is cleaned up to address only the config issue on #279

**Pull request closed without merging**